pkgs/by-name/e1/e1s/package.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "e1s";
+  version = "1.0.34";
+
+  src = fetchFromGitHub {
+    owner = "keidarcy";
+    repo = "e1s";
+    rev = "v${version}";
+    hash = "sha256-fxo7TQjFrp2BbXz7bwMC5AuacKpyi5UmAMEVBmR91ko=";
+  };
+
+
+  #vendorHash = "sha256-fxo7TQjFrp2BbXz7bwMC5AuacKpyi5UmAMEVBmR91ko=";
+  vendorHash = "sha256-pM6win2W6HqjIg9QBblDl3ptKEI6sANliXkpwlFnTb0=";
+
+  meta = with lib; {
+    description = "Easily Manage AWS ECS Resources in Terminal 🐱";
+    homepage = "https://github.com/keidarcy/e1s";
+    changelog = "https://github.com/derailed/e1s/releases/tag/v${version}";
+    license = licenses.mit;
+    mainProgram = "e1s";
+    maintainers = with maintainers; [ keidarcy mdb ezafeire ];
```

```suggestion
  meta = {
    changelog = "https://github.com/derailed/e1s/releases/tag/v${version}";
    description = "Easily Manage AWS ECS Resources in Terminal 🐱";
    homepage = "https://github.com/keidarcy/e1s";
    license = lib.licenses.mit;
    mainProgram = "e1s";
    maintainers = with maintainers; [
      keidarcy
      mdb
      ezafeire
    ];
```

- Using with lib; is discouraged, you can learn more about it here https://nix.dev/guides/best-practices.html#with-scopes and here https://github.com/NixOS/nixpkgs/pull/293767
- Sorted alphabetically

---

Just a note, none of these names are on the `maintainers/maintainer-list.nix`. Please read https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md on how to add yourself as a maintainer